src/notifications/api/serializers.py

```diff
@@ -156,11 +170,25 @@ def validate(self, attrs):
         validated_attrs = super().validate(attrs)
         # check if exchange exists
         try:
-            Kanaal.objects.get(naam=validated_attrs['kanaal'])
+            kanaal = Kanaal.objects.get(naam=validated_attrs['kanaal'])
         except ObjectDoesNotExist:
             raise serializers.ValidationError(
                 {'kanaal': _('Kanaal met deze naam bestaat niet.')},
                 code='message_kanaal')
+
+        # check if msg kenmerken are consistent with kanaal filters
+        kenmerken = validated_attrs['kenmerken']
+        if not kenmerken:
+            raise serializers.ValidationError(
+                {'kenmerken': _('Kenmerken should not be empty')},
```

Comment:
we defined kenmerken to vary from API to API though - so technically they can be empty if there's nothing relevant.

Reply:
removed that validation check